pkg/agent/agent.go

```diff
@@ -470,22 +476,7 @@ func (c *agentController) isHostedControlPlaneAvailable(status hyperv1alpha1.Hos
 	return false
 }
 
-func (c *agentController) CreateAddOnPlacementScore() {
-	listopts := &client.ListOptions{}
-	hcList := &hyperv1alpha1.HostedClusterList{}
-	err := c.spokeClient.List(context.TODO(), hcList, listopts)
-	if err != nil {
-		// just log the error. it should not stop the rest of reconcile
-		c.log.Error(err, "failed to get HostedCluster list")
-		return
-	}
-
-	items := []clusterv1alpha1.AddOnPlacementScoreItem{
-		{
-			Name:  util.HostedClusterScoresScoreName,
-			Value: int32(len(hcList.Items)),
-		},
-	}
+func (c *agentController) CreateAddOnPlacementScore(ctx context.Context) error {
```

Comment:
Very minor nit: `CreateAddOnPlacementScore` maybe should be renamed to `SyncAddOnPlacementScore`

Reply:
Done Thank you